src/liballoc/rc.rs

```diff
@@ -1436,6 +1436,29 @@ impl<T: ?Sized> fmt::Pointer for Rc<T> {
     }
 }
 
+#[unstable(feature = "rc_fn", issue = "none")]
```

Comment:
Unstable doesn't really work here -- implementing a stable trait for stable types is instantly stable too.

Reply:
I see that a lot of other trait impls have `#[stable(...)]`, should I remove this or just mark it `#[stable]` with the name and `since = "1.45"`?

Reply:
Updated to mark it stable.